test/e2e/density.go

```diff
@@ -141,17 +135,14 @@ var _ = Describe("Density", func() {
 			c.Pods(ns).Delete(name, nil)
 		}
 
-		By(fmt.Sprintf("Destroying namespace for this suite %v", ns))
-		if err := c.Namespaces().Delete(ns); err != nil {
-			Failf("Couldn't delete ns %s", err)
-		}
-
 		expectNoError(writePerfData(c, fmt.Sprintf(testContext.OutputDir+"/%s", uuid), "after"))
 
 		// Verify latency metrics
 		highLatencyRequests, err := HighLatencyRequests(c, 3*time.Second, sets.NewString("events"))
 		expectNoError(err)
 		Expect(highLatencyRequests).NotTo(BeNumerically(">", 0), "There should be no high-latency requests")
+
+		framework.afterEach()
```

Comment:
It's not that simple, because we're afraid of timeouts. On scalability cluster we need to delete 3000 pods, which takes some time, and I'm not certain how close to namespace deletion timeout we're running when deleting only events. I recall having this kind of problem with 'scheduler_predicates' test.